packages/arb-token-bridge-ui/src/hooks/useAddNewToken.ts

```diff
@@ -0,0 +1,55 @@
+import { useCallback, useState } from 'react'
```

Comment:
Can we comment in file why this hook is needed and what is the benefit of this being standalone?

Reply:
Reason are actually quite basic: extracting common logic